### DISCUSSION TITLE
Player not getting tracked after removing and triggering "livemap:AddPlayerData"

### DISCUSSION THREAD

**Discussion by RobijnGit** [TGRHavoc/live_map#62] [Q&A] [answered]
In the [document](https://docs.tgrhavoc.co.uk/livemap-resource/#client-to-server) it says 
> Note, when using `livemap:AddPlayerData` or `livemap:UpdatePlayerData` if the player has been removed using `livemap:RemovePlayer` they will be tracked again.

But, when I use `livemap:RemovePlayer` to remove the player, it wont show the player when I use `livemap:AddPlayerData` or `livemap:UpdatePlayerData`..

```
RegisterCommand("hide", function()
    print("hide")
    TriggerServerEvent("livemap:RemovePlayer")
end)

RegisterCommand("show", function()
    print("show")
    TriggerServerEvent("livemap:AddPlayerData", "", "")
end)
```

Is this something I did wrong or an issue?
Any help would be appreciated.

**Comment by TGRHavoc** [comment, marked as answer]
> TriggerServerEvent("livemap:RemovePlayer")

This removes all the data knows about the player, including his position, icon etc. So, you will need to add them back to the server.

Something like this in your custom client should work:
```
TriggerServerEvent("livemap:AddPlayerData", "pos", GetEntityCoords(PlayerPedId()))
TriggerServerEvent("livemap:AddPlayerData", "icon", 6)
```
I think this might be the least amount of data needed to show the blips. Note: You will need to update the pos data to see the blip move on the interface.

**Comment by TGRHavoc** [comment]
@RobijnGit did the last comment help? Am I able to close this issue? :)

**Comment by RobijnGit** [comment]
> @RobijnGit did the last comment help? Am I able to close this issue? :)

Yes it did, thank you.
